portfolio/src/main/webapp/index.html

```diff
@@ -75,7 +74,15 @@ <h1 class="title">
     </div>
     <div id="comments" class="content">
       <h2>Comments</h2>
-      <p>Click below to see comments!</p>
+      <p> I hope you enjoyed my portfolio! I would love to hear from you!
+          Please leave a comment below letting me know what you thought/
+          suggestions you may have for me!</p>
+      <form action="/data" method=POST>
+        <textarea name="comment">Comment...</textarea>
```

Comment:
Consider using the `placeholder` attribute as an alternative to setting an initial value.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea